neural_network/activation_functions/scaled_exponential_linear_unit.py

```diff
@@ -0,0 +1,44 @@
+"""
+Implements the Scaled Exponential Linear Unit or SELU function.
+The function takes a vector of K real numbers and two real numbers
+alpha(default = 1.6732) & lambda (default = 1.0507) as input and
+then applies the SELU function to each element of the vector.
+SELU is a self-normalizing activation function. It is a variant
+of the ELU. The main advantage of SELU is that we can be sure
+that the output will always be standardized due to its
+self-normalizing behavior. That means there is no need to
+include Batch-Normalization layers.
+References :
+https://iq.opengenus.org/scaled-exponential-linear-unit/
+"""
+
+import numpy as np
+
+
+def scaled_exponential_linear_unit(
+    vector: np.ndarray, alpha: float = 1.6732, _lambda: float = 1.0507
```

Comment:
Is there a reason why `_lambda` has an underscore? Is the user not meant to change this coefficient?

Reply:
Yes, firstly lambda is a reserved keyword in python. Secondly both alpha and lambda are fixed constants.

Reply:
The user can change these values, that may yield slightly different behaviour from the function, but the default values given to them are `alpha: float = 1.6732, _lambda: float = 1.0507`.

Reply:
Ah. Yes for sure, I'll get it done right away!